docs/main/concepts/basics.md

```diff
@@ -70,32 +68,32 @@ There are primarily two types of users that interact with Kubernetes and OpenEBS
 
 ### Cluster Administrators 
 
-These users are responsible for managing the life-cycle of the cluster and are often referred to as administrators or platform SREs. Administrators have full access to the cluster resources and can create policies and roles for other users that have access to the cluster.  
+These users are responsible for managing the life cycle of the cluster and are often referred to as administrators or platform SREs. Administrators have full access to the cluster resources and can create policies and roles for other users that have access to the cluster.
 
-Cluster administrators are responsible for installing OpenEBS and configuring the OpenEBS Storage Classes that will be used by other application users. 
+Cluster administrators are responsible for installing OpenEBS and configuring the OpenEBS Storage Classes that will be used by other application users.
 
 ### Application Owners 
 
-These users are responsible for managing the life-cycle of the application and are often referred to as users or developers. Typically these users have restricted access to one or more `namespaces` in the Kubernetes clusters. These users can have view access to the object that can use like `StorageClasses` but will not be able to edit or delete them. 
+These users are responsible for managing the life cycle of the application and are often referred to as users or developers. Typically these users have restricted access to one or more `namespaces` in the Kubernetes clusters. These users can have view access to the object that can be used like `StorageClasses` but will not be able to edit or delete them. 
```

Comment:
life cycle is not a single word?

Reply:
When referring to a single concept, we use a single or closed word, such as "lifecycle". In contrast, when referring to independent concepts, the open usage would be more acceptable.